docker-compose.yml

```diff
@@ -0,0 +1,34 @@
+version: '3.9'
```

Comment:
I'm pretty sure "version" is obsolete and can be removed.  Additionally, the new idiom is to call this file `compose.yaml`. Would you mind making that change?